src/test/java/mate/academy/jvbookstore/controller/OrderControllerTest.java

```diff
@@ -0,0 +1,318 @@
+        
+        ownersShoppingCart.setCartItems(new HashSet<>());
+        ownersShoppingCart.getCartItems().add(cartItem1);
+        ownersShoppingCart.getCartItems().add(cartItem2);
+        shoppingCartRepository.save(ownersShoppingCart);
+
+        PlaceOrderRequestDto requestDto = new PlaceOrderRequestDto();
+        requestDto.setShippingAddress("SA1");
+
+        MvcResult result = mockMvc.perform(post("/orders")
+                .content(objectMapper.writeValueAsString(requestDto))
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andReturn();
+
+        OrderDto actual = objectMapper.readValue(
+                result.getResponse().getContentAsString(), OrderDto.class);
+
+        Assertions.assertNotNull(actual);
+        Assertions.assertNotNull(actual.getId());
+        Assertions.assertEquals(ownersShoppingCart.getUser().getId(), actual.getUserId());
+        Assertions.assertTrue(actual.getOrderDate().isBefore(LocalDateTime.now()));
+        Assertions.assertEquals(Order.Status.CREATED, actual.getStatus());
+        Assertions.assertEquals(2, actual.getOrderItems().size());
+        BigDecimal expectedTotalPrice = BigDecimal.ZERO;      
+        expectedTotalPrice = expectedTotalPrice.add(cartItem1.getBook()
+                .getPrice()
+                .multiply(BigDecimal.valueOf(cartItem1.getQuantity())))
+                .add(cartItem2.getBook()
+                .getPrice()
+                .multiply(BigDecimal.valueOf(cartItem2.getQuantity())));      
+        Assertions.assertEquals(0, expectedTotalPrice.compareTo(actual.getTotal()));
+    }
+
+    @Test
+    @WithUserDetails(value = "owner@bookstore.com")
+    void getOrders_WithPageable_ListOfOneOrder(
+            @Autowired OrderRepository orderRepository) throws Exception {
+        Order expected = addOrderToDb(orderRepository);
+
+        Pageable pageable = PageRequest.of(0, 10);
+
+        MvcResult result = mockMvc.perform(get("/orders")
+                .content(objectMapper.writeValueAsString(pageable))
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andReturn();
+
+        OrderDto[] actual = objectMapper.readValue(
+                result.getResponse().getContentAsString(), OrderDto[].class);
+
+        Assertions.assertNotNull(actual);
+        Assertions.assertEquals(1, actual.length);
+        Assertions.assertEquals(expected.getId(), actual[0].getId());
+        Assertions.assertEquals(expected.getUser().getId(), actual[0].getUserId());
+        Assertions.assertEquals(expected.getStatus(), actual[0].getStatus());
+        Assertions.assertEquals(expected.getTotal(), actual[0].getTotal());
+        Assertions.assertEquals(2, actual[0].getOrderItems().size());
+        Assertions.assertEquals(new HashSet<>(getOrderItemDtos(expected)),
+                actual[0].getOrderItems());
```

Comment:
avoid using multiple assertEquals() for comparing two objects fields, instead implement .equals() method for these objects and check only whole objects via single assertEquals() method